### DISCUSSION TITLE
Docker Compose Error

### DISCUSSION THREAD

**Discussion by MultivacX** [langgenius/dify#5150] [General]
`
docker compose version
Docker Compose version v2.27.0-desktop.2

docker compose up -d

[+] Running 9/9
 ✘ sandbox Error    context canceled                                       2.5s 
 ✘ ssrf_proxy Error context canceled                                       2.5s 
 ✘ worker Error     context canceled                                       2.5s 
 ✘ web Error        context canceled                                       2.5s 
 ✘ api Error        Head "https://registry-1.docker.io/v2/la...            2.5s 
 ✘ nginx Error      context canceled                                       2.5s 
 ✘ redis Error      context canceled                                       2.5s 
 ✘ db Error         context canceled                                       2.5s 
 ✘ weaviate Error   context canceled                                       2.5s 
Error response from daemon: Head "https://registry-1.docker.io/v2/langgenius/dify-api/manifests/0.6.10": unauthorized: incorrect username or password
`

**Comment by MultivacX** [comment]
Try first:

$ docker logout

**Comment by xiaopailang** [comment]
hello， Have you solved this problem
